utils/reviewsUtils.js

```diff
@@ -1,15 +1,51 @@
 import { API_BASE } from "../routes/api.js";
 import fetch from "node-fetch";
 
-
-//Get reviews for each movie and select page
-export async function getReviewsSizeFive(id, page){
-  const res = await fetch(`${API_BASE}/reviews?filters[movie]=${id}&pagination[pageSize]=5&pagination[page]=${page}`)
+/*Fetches all reviews for each movie from CMS then filters the data to remove
+all unverified or invalid reviews. Then the function paginates the array and sets 
+the pagesize to 5*/
+export async function getReviewsSizeFive(id, page) {
+  const res = await fetch(`${API_BASE}/reviews?filters[movie]=${id}`)
   const payload = await res.json();
-    return payload.data.map((obj) => ({
+
+  const modifiedArr = payload.data.map((obj) => ({
     id: obj.id,
     ...obj.attributes
-  }));
+  }))
+
+  let filteredArr = filterVerified(modifiedArr);
+
+  filteredArr.forEach(function (obj) {
+    obj.author === null ? obj.author = 'Okänd användare' : obj.author = obj.author;
+    obj.author === '' ? obj.author = 'Okänd användare' : obj.author = obj.author;
```

Comment:
There are two problems with this piece of code in my opinion.

The biggest problem is that you're using ternary expressions to _execute code_ in the expression-if-true and expression-if-false parts of the ternary. That is very uncommon, and produces hard to read and hard to understand code.

The point of a ternary expression is that it will evaluate to something, as opposed to if statements that don't evaluate to anything (they just control the flow of the code). So you should probably do this instead:

```js
// Assignment target = condition ? expression-if-true : expression-if-false;
obj.author = obj.author === null ? 'Okänd användare' : obj.author;
```

The next problem is the duplication. Instead of having almost the exact same code twice, you could check if `author` is [falsy](https://developer.mozilla.org/en-US/docs/Glossary/Falsy), and not just two specific values. Falsy means that a non-boolean value (e.g. a string or something else) is understood as false when interpreted as a boolean. Both `null` and `''` (the empty string) are falsy. So the following code would work:

```js
obj.author = !obj.author? 'Okänd användare' : obj.author;
```

But that makes me wonder, why is this a ternary expression to begin with, if only one of the branches (when `obj.author` is falsy) results in any changes. I would just use an if statement instead:

```
if (!obj.author) {
  obj.author = 'Okänd användare';
}
```

Or, if you really want a single line expression, you can use the `||` operator. It's a fun operator, that doesn't just mean "OR", but actually evaluates to the left value if it's truthy, or the right value if the left one is falsy. So you can do this:

```
// Assignment target = x-if-truthy || value-if-x-is-falsy
obj.author = obj.author || 'Okänd användare';
```

And in fact, just like `x = x + y` can also be expressed `x += y`, the expression `x = x || y` is synonymous with `x ||= y`, so the shortest way of solving this is actually:

```js
obj.author ||= 'Okänd användare';
```

I've explained the underlying logic for this operator above, but I tend to just think of the `||=` operator as a special "assign only if falsy" operator.